File: C#/Lab1.2/Applicants.cs
using System;

namespace lab1._2
{
    public class Applicants
    {
        public string full_name;
        public string birth;
        public string sex;

        public Applicants(string full_name, string birth, string sex)
        {
            this.full_name = full_name;
            this.birth = birth;
            this.sex = sex;
        }

        public void Output()
        {
            Console.WriteLine($"Full name: {full_name}, birth: {birth}, sex: {sex}");
        }
    }
}

File: Python/Lab1.2/applicants.py
class Applicants:
    def __init__(self, full_name, birth, sex):
        self.full_name = full_name
        self.birth = birth
        self.sex = sex

    def Output(self):
        print(f"Full name: {self.full_name}, birth: {self.birth}, sex: {self.sex}")